params/config.go

```diff
@@ -560,7 +560,7 @@ type ChainConfig struct {
 	// Miko hardfork introduces sponsored transactions
 	MikoBlock   *big.Int `json:"mikoBlock,omitempty"`   // Miko switch block (nil = no fork, 0 = already on activated)
 	TrippBlock  *big.Int `json:"trippBlock,omitempty"`  // Tripp switch block (nil = no fork, 0 = already on activated)
-	TrippPeriod *big.Int `json:"trippPeriod,omitempty"` // The period number of Tripp block
+	TrippPeriod *big.Int `json:"trippPeriod,omitempty"` // The period number when Tripp consensus rule is effective.
```

Comment:
This is not the number of period where Tripp is effective, this is the period number that Tripp block happens. The period number where Tripp is effective is this number + 1.